core/zms/src/main/rdl/Authority.rdli

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 Athenz Authors
+// Licensed under the terms of the Apache version 2.0 license. See LICENSE file for terms.
+
+//List of valid user authority attribute values
+type UserAuthorityValidValuesList Struct {
+    Array<String> validValues; //list of valid values for attribute
+}
+
+//List all valid values for the user authority
+resource UserAuthorityValidValuesList GET "/authority/user?attributeType={attributeType}" {
```

Comment:
let's use /authority/user/attribute?type={type} in case we need to add more endpoints for the user authority in the future

Reply:
Endpoint now return all types so changed it to:
/authority/user/attribute